py/makeqstrdefs.py

```diff
@@ -86,7 +86,8 @@ def write_out(fname, output):
 
 
 def process_file(f):
-    re_line = re.compile(r"#[line]*\s\d+\s\"([^\"]+)\"")
+    # match gcc-like output (# n "file") and msvc-like output (#line n "file")
+    re_line = re.compile(r"^#(line)?\s+\d+\s\"([^\"]+)\"")
```

Comment:
Since `line` is not important, why not make it a non-capturing group?
```suggestion
    re_line = re.compile(r"^#(?:line)?\s+\d+\s\"([^\"]+)\"")
```

Reply:
Missed that feature in Python docs! Thanks, updated.